docs/rules/google_project_iam_invalid_member_format.md

```diff
@@ -1,4 +1,4 @@
-# google_project_iam_member_invalid_member_format
+# google_project_iam_invalid_member_format
```

Comment:
Why is this renamed? This may have unintended changes as the `google_project_iam_member_invalid_member_format` rule will return a link to `docs/rule/google_project_iam_member_invalid_member_format.md`.

Personally, I recommend that this kind of rule be `${resource_name}_invalid_${attribute_name}`. "member" is duplicated in the name, but I think it is more important to be able to identify the resource to be inspected from the rule name.